code/modules/integrated_electronics/passive/power.dm

```diff
@@ -66,13 +66,18 @@
 // For implants.
 /obj/item/integrated_circuit/passive/power/metabolic_siphon
 	name = "metabolic siphon"
-	desc = "A complicated piece of technology which converts bodily nutriments of a host into electricity."
+	desc = "A complicated piece of technology which converts bodily nutriments of a host into electricity, or vice versa."
```

Comment:
I am not sure about the energy -> nuriments part as this implies the technology to generate matter from energy exists and is small enough to be fitted into a circut (which afaik doesnt even exist in our lore)

Reply:
Noooo. No. That's wroooong. It takes the principle of ATP synthase, using a proton gradient to drive an enzyme that converts ADP to ATP (cellular chemical energy) and uses electricity to run it instead. Something like genetically modified immobilised ATP synthase enzymes that run off of electricity instead of proton gradients would probably be possible even today, and no doubt with the technology we have in our lore.
There is NO matter generation. It is (rather inefficiently) creating cellular energy by using electricity.